### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by DT1000101**
add requirements.
remove forced 6 test images.

### CODE REVIEW

File: .gitignore
Comment (HasnainRaz) [inline .gitignore]: Add an empty line pls

File: infer.py
Comment (HasnainRaz) [inline infer.py:34]: This is not ideal if the number of images is 1000....on one plot, it will never be viewable or fit.

**Review by HasnainRaz — CHANGES_REQUESTED** [review]
Thanks for the requirements file, though it looks like there are unnecessary packages in it. I'll merge it if you can thin it out to the essentials and address the review comments.

File: infer.py
Comment (DT1000101) [reply infer.py:34]: Just thought it made more sense than forcing the folder to have exactly 6 images. changed it to:
```nr_images = len(image_paths) if len(image_paths) < 6 else 6```
so that it works for <=6 and if there are more than 6 then it caps it off.

**Comment by DT1000101** [comment]
> 

Regarding the unnecessary packages, they are just what were installed as co-dependencies of the required packages, and then they were added automatically when I did `pip > freeze`.

**Review by HasnainRaz — APPROVED** [review]
LGTM

**Comment by DT1000101** [comment]
@HasnainRaz nice :) I can't merge it btw (no write access), if you want it to be added you would have to merge it :)

**Comment by HasnainRaz** [comment]
> @HasnainRaz nice :) I can't merge it btw (no write access), if you want it to be added you would have to merge it :)

Whoops, didnt realize that

**Pull request merged**
